File: baekjoon_java/MakeOne.java
package baekjoon_java;
import java.util.*;
public class MakeOne {
	static int fun(int number) {
		int sum = 0;
		for(int i = 0; number > 0 ; i++) {
			sum += number % 10;
			number = number / 10;
		}
		if(sum < 10) {
			return sum;
		}
		else {
			return fun(sum);
		}
	}
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int num = input.nextInt();
		int result = fun(num);
		System.out.println(result);
	}
}

File: baekjoon_python/MakeOne.py
def fun(number):
    sum = 0
    for i in number:
        sum += int(i)

    if len(str(sum)) == 1:
        return sum
    elif len(str(sum)) >= 2:
        r_sum = 0
        for j in str(sum):
            r_sum += int(j)

        return r_sum

number = input("Enter the number : ")
print(fun(number))